src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -281,7 +281,7 @@ public int Count
         {
             get
             {
-                VersionCheck();
+                VersionCheckCount(true);
```

Comment:
One last thing, sorry for the back and forth iterations, but for bool parameters we like to name them so that when you read this statement is clear and you don't have to go to the method declaration to see what that true stands for, i.e: 

`VersionCheck(updateCount: true);`

and just noticed you're calling VersionCheckCount still, it should now be VersionCheck. 

Also, would you mind re running your benchmarks just to make sure we're still getting the perf benefits?

Reply:
Fixed that above issue and sure, I'll rerun the benchmarks.

Reply:
The performance benefits are still there. Tested for Set of size 5000000 and SubSet (View) of Size 500000.

Reply:
Nice! Thanks for the update!